simcon_project/simcon_project/admin.py

```diff
@@ -1,9 +0,0 @@
-from django.contrib import admin
-from django.apps import apps
-from simcon_project.conversation_templates.models import *
-
-admin.site.register(ConversationTemplate)
-admin.site.register(TemplateNode)
-admin.site.register(TemplateNodeChoice)
-admin.site.register(TemplateResponse)
-admin.site.register(TemplateNodeResponse)
```

Comment:
We should keep this in until the backend is setup

Reply:
When I checked, it made no difference whether this was there or not. I believe the loop inside users admin.py registers every model. I can double check though.

Reply:
Oh I didn't know that. As long as we can see it on admin then we should remove it!